### PR TITLE
Migrations, Seed, Models and GraphQL

### PR DESCRIPTION
**Pull request by nazar**
The Dockerfile should now run migrations on container startup - so schemas will always be upto date.

Run `yarn seed:load` inside the container to load seed data.

The Graphiql query tool is exposed on development env on http://localhost:3001

Added a basic table `campaigns` with corresponding graphSQL schema definition, resolvers and queries to query all campaigns or a specific one - ie

```
{
  campaigns{
    id
    title
    gameSettings{
      minPlayers
      maxPlayers
      skillLevel
      postingFrequency
    }
  }
}
```

and 

```
{
  campaign(id: 1){
    id
    title
    gameSettings{
      minPlayers
      maxPlayers
      skillLevel
      postingFrequency
    }
  }
}
```

### CODE REVIEW

**Comment by nazar** [comment]
merge conflict resolved @mikelax

**Comment by nazar** [comment]
@mikelax - I just pushed another tiny tweak to replace `node.wheezy` with the alpine version.

I _think_ you'll need to use `docker-compose up --build` the first time to rebuild the containers.

**Comment by mikelax** [comment]
👍 @nazar. looks good, just one question on managing the config values. 
I created a local-development.json file with attributes:
```js
{
  "auth0": {
    "clientId": "value",
    "domain": "value"
  }
}
```

How would someone (easily) know that this file needs to be created and with these attributes? Should we add *every* attribute to the `default.json` file and just have empty strings with the idea they will be provided in higher config files?

**Comment by nazar** [comment]
Hey @mikelax,

>How would someone (easily) know that this file needs to be created and with these attributes? 

I was thinking about that too. I do like the method we use in Match with secret.js.sample. We could add a similar file in the api call `local-development.json.sample` with required keys - and reference that in the README

>Should we add every attribute to the default.json file and just have empty strings with the idea they will be provided in higher config files?

It's always tricky deciding what goes in the `default.json` file. I try and keep env wide configs in there that are relevant to almost all environments but can then be overridden in any of the specific `envs`. I'm not too fond of adding empty strings to it - maybe a README file inside the `config` folder is a another way of documenting configs?

File: api/app.js [hunk 1]
```diff
@@ -48,8 +48,8 @@ if (config.get('graphql.graphiql')) {
 // set up basic routes
 app.use('/silent', (req, res) => {
   res.render('pages/silent', { 
-    clientID: process.env.REACT_APP_AUTH0_CLIENT_ID,
-    domain: 'forgingadventures.auth0.com',
+    clientID: config.get('auth0.clientId'),
+    domain: config.get('auth0.domain'),
     redirectUri: 'http://localhost:3000'
```
Comment (nazar) [inline api/app.js]: can `redirectUri` go in the `development.json` file? i.e.

```
"auth0": {
  "redirectUri": "http://localhost:3000"
}
```

Comment (mikelax) [reply api/app.js]: Yeah it should. Not sure why I moved two strings but not the third.

Comment (nazar) [reply api/app.js]: I'll add it there - couldn't resist adding a few more things to the PR :homerdisappear:

**Review by mikelax — APPROVED** [review]
🚀

**Pull request merged**